src/protocol/LSP.Capabilities.pas

```diff
@@ -164,26 +332,70 @@ constructor TServerCapabilities.Create(settings: TServerSettings);
   implementationProvider := true;
   referencesProvider := true;
   documentHighlightProvider := true;
-  executeCommandProvider := TExecuteCommandOptions.Create([
+  fexecuteCommandProvider := TExecuteCommandOptions.Create([
     'pasls.completeCode'
   ]);
-  inlayHintProvider := nil;//TInlayHintOptions.Create;
+  finlayHintProvider:= TInlayHintOptions.Create;
```

Comment:
Keep this commented out. Inlay hints are not implemented right now (not sure they're possible even with Code Tools)

Reply:
Done.